<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PDAS" failures="0" tests="1" name="TestNgDemo.DataProviderExample" time="0.000" errors="1" timestamp="2024-12-05T12:06:26 IST" skipped="0">
  <testcase classname="TestNgDemo.DataProviderExample" name="test1" time="0.000">
    <error message="
Some DataProvider public java.lang.Object[][] TestNgDemo.DataProviderExample.getData1(com.sun.org.apache.bcel.internal.classfile.Method) parameters unresolved:  at 0 type class com.sun.org.apache.bcel.internal.classfile.Method
" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Some DataProvider public java.lang.Object[][] TestNgDemo.DataProviderExample.getData1(com.sun.org.apache.bcel.internal.classfile.Method) parameters unresolved:  at 0 type class com.sun.org.apache.bcel.internal.classfile.Method

at org.testng.internal.invokers.MethodInvocationHelper.getParameters(MethodInvocationHelper.java:239)
at org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:158)
at org.testng.internal.Parameters.handleParameters(Parameters.java:809)
at org.testng.internal.Parameters.handleParameters(Parameters.java:734)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:946)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- test1 -->
</testsuite> <!-- TestNgDemo.DataProviderExample -->
